README.md

```diff
@@ -1,60 +1,100 @@
 <div align="center">
-<h1>interchaintest</h1>
+<h1><code>interchaintest</code></h1>
 
 Formerly known as `ibctest`.
 
 [![Go Reference](https://pkg.go.dev/badge/github.com/strangelove-ventures/interchaintest@main.svg)](https://pkg.go.dev/github.com/strangelove-ventures/interchaintest@main)
 [![License: Apache-2.0](https://img.shields.io/github/license/strangelove-ventures/interchaintest.svg?style=flat-square)](https://github.com/strangelove-ventures/interchaintest/blob/main/create-test-readme/LICENSE)
 [![Go Report Card](https://goreportcard.com/badge/github.com/strangelove-ventures/interchaintest)](https://goreportcard.com/report/github.com/strangelove-ventures/interchaintest)
+[![Conforms to README.lint](https://img.shields.io/badge/README.lint-conforming-brightgreen)](https://github.com/strangelove-ventures/readme-dot-lint)
+</div>
+
+
+
+🌌 Why use `interchaintest`?
+=============================
+
+In order to ship production-grade software for the Interchain, we needed sophisticated developer tooling...but IBC and Web3 have a *lot* of moving parts, which can lead to a steep learning curve and all sorts of pain. Recognize any of these?
+
+- repeatedly building repo-specific, Docker- and shell-based testing solutions,
+- duplication of effort, and
+- difficulty in repurposing existing testing harnesses for new problem domains.
 
+We built `interchaintest` to extract patterns and create a generic test harness: a use-case-agnostic framework for generating repeatable, diagnostic tests for every aspect of IBC.
 
+Read more at the [Announcing `interchaintest` blog post](https://strange.love/blog/announcing-interchaintest).
 
-`interchaintest` orchestrates Go tests that utilize Docker containers for multiple
+🌌🌌 Who benefits from `interchaintest`?
+=============================
+
+`interchaintest` is for developers who expect top-shelf testing tools when working in IBC.
+
+
+🌌🌌🌌 What does `interchaintest` do?
+=============================
+
+`interchaintest` is a framework for testing blockchain functionality and interoperability between chains, primarily with the Inter-Blockchain Communication (IBC) protocol.
+
+Want to quickly spin up custom testnets and dev environments to test IBC, chain infrastructures, smart contracts, etc.? `interchaintest` orchestrates Go tests that utilize Docker containers for multiple
```

Comment:
We should mention how interchaintest can automatically create clients, connections, channels and then spin up a relayer. 

I realize this gets grouped into "IBC" but I think this is one of its most under looked features.

Reply:
generalized this to ibc Relayer setup with a link to the cosmos/relayer repo